lib/main.coffee

```diff
@@ -54,6 +54,14 @@ module.exports =
       items:
         type: 'boolean'
       order: 7
+    hidechars:
+      title: 'Hide character count'
+      description: 'Hides the character count from the view'
+      type: 'boolean'
+      default: 'false'
+      items:
+        type: 'boolean'
```

Comment:
It's just a monkey see monkey do solution I copied from the previous boolean checkbox. It seems not to matter for atom in the end, it works with and without this `items`.